proto/viam/app/v1/robot.proto

```diff
@@ -134,6 +135,7 @@ message Frame {
   string parent = 1;
   Translation translation = 2;
   Orientation orientation = 3;
+  common.v1.Geometry geometry = 4;
```

Comment:
> Frame system decomposition into proto messages currently simply ignores geometry information. This will be updated to include geometry information.

It was mentioned in text, but the proto message itself was not explicitly mentioned.

This is separate from the other changes we've been discussing, as it's part of the App API, not the RDK API.

Information is captured here: https://viam.atlassian.net/browse/DATA-874

Adding this was missed when Geometries were added to the Viam cloud app